addon/components/pikaday-input.js

```diff
@@ -35,7 +37,12 @@ export default Ember.Component.extend({
   }.on('willDestroyElement'),
 
   userSelectedDate: function() {
-    this.set('value', this.get('pikaday').getDate());
+    var selectedDate = this.get('pikaday').getDate();
+    if (this.get('utc'))
```

Comment:
I would like to name the attribute `useUTC` since it makes it more obvious what it does. In addition the `{` needs to be on the same line as the `if`.